packages/imhex.vm/imhex.vm.nuspec

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>imhex.vm</id>
+    <version>1.31.0.20231018</version>
+    <authors>WerWolv</authors>
+    <description>A Hex Editor for Reverse Engineers, Programmers and people who value their retinas when working at 3 AM.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="imhex" version="1.31.0" />
```

```suggestion
      <dependency id="imhex" version="[1.31.0]" />
```

`version="1.31.0"` requires >=1.31.0, while `version="[1.31.0]"` requires exactly version `1.31`. See nuspec documentation for more details: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges

We want to require a concrete version to ensure that we have tested that the version installs correctly and we update the version automatically once a week.  This is documented in the [Wiki](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#dependencies).

This is the way it is done if you use the metapackage template (by using `--type METAPACKAGE`): https://github.com/mandiant/VM-Packages/blob/main/scripts/utils/create_package_template.py#L66 😉

---

Solved at https://github.com/mandiant/VM-Packages/pull/700/commits/a0a79e9ca926a7158a5e9f988ecc35b328e43dab.